Wikipedia/Code/WMFTextualSaveButton.m

```diff
@@ -0,0 +1,132 @@
+//
+//  WMFTextualSaveButton.m
+//  Wikipedia
+//
+//  Created by Brian Gerstle on 1/12/16.
+//  Copyright © 2016 Wikimedia Foundation. All rights reserved.
+//
+
+#import "WMFTextualSaveButton.h"
+#import <Masonry/Masonry.h>
+#import "UIColor+WMFStyle.h"
+
+@implementation WMFTextualSaveButton
+
+- (instancetype)initWithCoder:(NSCoder*)aDecoder {
+    self = [super initWithCoder:aDecoder];
+    if (self) {
+        [self commonInit];
+    }
+    return self;
+}
+
+- (instancetype)initWithFrame:(CGRect)frame {
+    self = [super initWithFrame:frame];
+    if (self) {
+        [self commonInit];
+    }
+    return self;
+}
+
+- (void)commonInit {
+    [self setupViews];
+    [self applyInitialState];
+}
+
+#pragma mark - Accessors
+
+- (UIImage*)iconImage {
+    return self.selected ? ([UIImage imageNamed : @"save-filled-mini"]) : ([UIImage imageNamed:@"save-mini"]);
+}
+
+- (NSString*)labelText {
+    return self.selected ? MWLocalizedString(@"button-saved-for-later", nil) : MWLocalizedString(@"button-save-for-later", nil);
+}
+
+#pragma mark - View Setup
+
+- (void)setupViews {
+    [self addMissingViews];
+
+    // subviews always follow super's tint color
+    self.saveIconImageView.tintAdjustmentMode = UIViewTintAdjustmentModeAutomatic;
+    self.saveTextLabel.tintAdjustmentMode     = UIViewTintAdjustmentModeAutomatic;
+}
+
+/**
+ *  Add any views which weren't set to the receiver's IBOutlet's in interface builder.
+ *
+ *  This allows the component to be created programmatically, dropped into IB using default views, and customizing one
+ *  or more subviews.
+ */
+- (void)addMissingViews {
```

Comment:
Yeah, YAGNI's probably the safer bet at this point.  This is meant to be a standard save button (i.e. effectively hard coding the label text & icon image), so customizing doesn't really make sense.

Regarding IBDesignable, I'm tempted to implement it anyway since it's already being used in IB—even if its outlets aren't assigned to anything.